src/ducks/_Table/index.js

```diff
@@ -39,40 +38,50 @@ const Table = ({
   itemProps,
   isLoading,
   getItemKey,
-  onRowClick
+  onRowClick,
+  isWithColumnTitles = true,
+  emptySection,
+  ...props
 }) => (
   <table className={cx(styles.wrapper, className)}>
-    <thead>
-      <tr>
-        {columns.map(({ id, title, Title }) => (
-          <th key={id}>{Title ? <Title {...itemProps} /> : title}</th>
-        ))}
-      </tr>
-    </thead>
+    {isWithColumnTitles && (
+      <thead>
+        <tr>
+          {columns.map(({ id, title, Title }) => (
+            <th key={id}>{Title ? <Title {...itemProps} /> : title}</th>
+          ))}
+        </tr>
+      </thead>
+    )}
     <tbody>
-      {items.map((item, i) => {
-        const itemIndex = offset + i
-
-        return (
-          <tr
-            key={getItemKey ? getItemKey(item) : item[itemKeyProperty]}
-            onClick={onRowClick && (e => onRowClick(item, e))}
-          >
-            {columns.map(({ id, render, className }) => (
-              <td key={id} className={className}>
-                {render(item, itemProps, itemIndex)}
-              </td>
-            ))}
-          </tr>
-        )
-      })}
-      {minRowsPadding(minRows, columns, items)}
+      {!isLoading && items.length === 0 && emptySection && (
+        <tr className={styles.disableHover}>
+          <td colSpan={columns.length}>{emptySection}</td>
+        </tr>
+      )}
```

Comment:
This breaks the render of the components that are using `Table` component. Keep the empty list case as it was before with adjustment to custom element:
```js
<caption>
       <Skeleton show={isLoading} className={styles.skeleton} />
       {!isLoading && items.length === 0 && (
         {emptySection || <NoDataImage className={styles.nodata} />}
       )}
</caption>
```